app/_src/introduction/concepts.md

```diff
@@ -71,3 +71,17 @@ They can enable traffic management, security, observability and traffic reliabil
 
 Policies always have a clear specific area of impact and goal.
 To learn more about [policies checkout the in depth introduction](/docs/{{ page.release }}/policies/introduction).
+
+### Targets (targetRef)
+
+A **Target** (referenced by `targetRef`) is a way to specify exactly which destination resource or workload a policy or routing rule should apply to. Rather than matching all services or inbounds by name or tag, `targetRef` allows fine-grained targeting of:
+
+- A specific **workload instance** (e.g., a single Dataplane proxy)  
+- A **subset** of inbounds/endpoints for a service
```

Comment:
If I understand correctly, you are taking here about destination service. Using name inbound in this case is not the best option as we define `Inbound` as a part of a dataplane proxy. In this case, I think it would be best to call it `service port`.